### DISCUSSION TITLE
Wifi not working

### DISCUSSION THREAD

**Discussion by NyRoo12** [elementary/triage#558] [Q&A]
A couple of days ago I installed elementary OS on my laptop, but when I install it, I don't get the option to connect to a wifi network

![Captura realizada el 2024-08-20 17 25 51](https://github.com/user-attachments/assets/3c3e32cf-cda0-4274-9d9b-c07d6ea7a321)

**Comment by vintoquel** [comment]
Hola, aqui dice que escriba una respuesta pero soy nuevo en esto. A mi tambien me pasa lo mismo, no me aparece el wifi y solo dice modo avion. Solo funciona el internet con el cableado al celular. ¿encontraste alguna solución?
Did you find a solution?
